src/development/LocalTest/Controllers/Storage/InstancesController.cs

```diff
@@ -713,6 +713,11 @@ private async Task<(Application, ActionResult)> GetApplicationOrErrorAsync(strin
                 errorResult = StatusCode(500, $"Unable to perform request: {e}");
             }
 
+            if (appInfo == null && errorResult == null)
```

Comment:
Not sure where this comes from, I have not added this code

Reply:
Might have merged a branch that someone else ont he OED team has worked with previously? I would suggest you just undo the change if it wasn't intentional.

Reply:
Will find out where it came from and undo it if it doesnt make sense, thanks :)

Reply:
Removed it as I suspect its from an old commit which has since been removed considering it is not in master already